packages/travix-ui-kit/components/selectionBlock/selectionBlock.js

```diff
@@ -96,15 +96,15 @@ SelectionBlock.propTypes = {
   /**
    * The logo label.
    */
-  logoLabel: PropTypes.string,
+  logoLabel: PropTypes.node,
   /**
    * The selection block subtitle.
    */
-  subtitle: PropTypes.string,
+  subtitle: PropTypes.node,
   /**
    * The selection block title.
    */
-  title: PropTypes.string,
+  title: PropTypes.node,
```

Comment:
Why not use `PropTypes.oneOfType([PropTypes.string, PropTypes.node])` and support both?

Reply:
@rbardini the node already contains these types: numbers, strings, elements and an array